skycoin/skywire
# Add package directive to Makefile

…d hypervisor

Did you run `make format && make check`?
No changes to the source were made outside the Makefile except adding systemd services. Possibly unnecessary for me to do that.

Partial fix for #387 

 Changes:	
- Add package directives to the Makefile for building amd64, arm64, and armhf .deb packages

How to test this PR:

Create packages for amd64, arm64, and armhf:
(use root so permissions are correct)
```
sudo make all-packages
```

install the correct package for your architecture (assuming you use amd64 debian):
```
sudo dpkg -i *amd64.deb
```

if you have configuration files in place at `/etc/skywire-viosor.json` or `/etc/skywire-hypervisor.json`, start the systemd service(s) of your choice and verify it's functionality

```
sudo systemctl start skywire-visor
sudo systemctl start skywire-hypervisor
```
